release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -175,7 +185,7 @@ private void OnApplicationFocus(bool hasFocus)
 
 	private IEnumerator InitializeAtEndOfFrame()
 	{
-		yield return new WaitForEndOfFrame();
+		yield return null;
```

Comment:
I would call this method something like `InitialzieOnNextFrame` now to be accurate.